sum-app.js

```diff
@@ -0,0 +1,50 @@
+class SumApp {
+  // Add you solution here
+  constructor() {
+    this.numbers = []
+  }
+
+  addNumber(num) {
+    if (isNaN(num)) {

```

Comment:
Super, dass du an die Prüfung denkst.

Kannst du in Zukunft gern weglassen --> TypeScript